src/http-response-message.js

```diff
@@ -3,60 +3,68 @@ import {Headers} from './headers';
 import {RequestMessage} from './request-message';
 
 export class HttpResponseMessage {
-  constructor(requestMessage : RequestMessage, xhr : XHR, responseType : string, reviver : Function){
-    this.requestMessage = requestMessage;
-    this.statusCode = xhr.status;
-    this.response = xhr.response || xhr.responseText;
-    this.isSuccess = xhr.status >= 200 && xhr.status < 400;
-    this.statusText = xhr.statusText;
-    this.reviver = reviver;
-    this.mimeType = null;
+  constructor(requestMessage : RequestMessage, xhr : XHR, responseType : string, reviver : Function) {
+    let me = this;
```

Comment:
I've seen this technique in other frameworks, ExtJS for instance, it helps the uglifier to minify the code. The gain is not big, perhaps it doesn't justify it.